README.md

```diff
@@ -78,18 +78,20 @@ Lightning is rigorously tested across multiple GPUs, TPUs CPUs and against major
+|  Linux py3.8 (with Conda   | [![PyTorch & Conda](https://github.com/PyTorchLightning/pytorch-lightning/workflows/PyTorch%20&%20Conda/badge.svg?branch=master&event=push)](https://github.com/PyTorchLightning/pytorch-lightning/actions?query=workflow%3A%22PyTorch+%26+Conda%22+branch%3Amaster) | [![PyTorch & Conda](https://github.com/PyTorchLightning/pytorch-lightning/workflows/PyTorch%20&%20Conda/badge.svg?branch=master&event=push)](https://github.com/PyTorchLightning/pytorch-lightning/actions?query=workflow%3A%22PyTorch+%26+Conda%22+branch%3Amaster) | [![PyTorch & Conda](https://github.com/PyTorchLightning/pytorch-lightning/workflows/PyTorch%20&%20Conda/badge.svg?branch=master&event=push)](https://github.com/PyTorchLightning/pytorch-lightning/actions?query=workflow%3A%22PyTorch+%26+Conda%22+branch%3Amaster) | [![PyTorch & Conda](https://github.com/PyTorchLightning/pytorch-lightning/workflows/PyTorch%20&%20Conda/badge.svg?branch=master&event=push)](https://github.com/PyTorchLightning/pytorch-lightning/actions?query=workflow%3A%22PyTorch+%26+Conda%22+branch%3Amaster) |
+|      Linux py3.{7,9}       |                                                                                                                                 TODO                                                                                                                                 |                                                                                                                                  -                                                                                                                                   |                                                                                                                                  -                                                                                                                                   |                                                                                                                                 TODO                                                                                                                                 |
```

Comment:
@Borda how do you get the links for these jobs?
Does this PR need to be merged first? (since the jobs won't have run until they are on master)

Reply:
the job has to run at least one on any repo branch, then in the job options
![image](https://user-images.githubusercontent.com/6035284/140890041-42a5a029-0bd1-45f4-8615-6ac85f945524.png)

Reply:
That means we get one badge per workflow, not one badge per job in the workflow, correct?